tests/excerptexport/migrations/test_0019_one_export_per_file_format.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+import pytest
+from hamcrest import assert_that, contains_inanyorder as contains_in_any_order
+
+
+@pytest.fixture
+def extraction_configuration_with_formats():
+    return {'gis_formats': ['txt', 'foobar'], 'gis_options': {'detail_level': 'standard'}}
+
+
+@pytest.mark.django_db
+def test_0019_forward(migrate, user, extraction_configuration_with_formats):
+    old_apps = migrate('excerptexport', '0017_auto_20160310_1102')
+    ExtractionOrder = old_apps.get_model('excerptexport', 'ExtractionOrder')  # noqa
+    extraction_order = ExtractionOrder.objects.create(orderer_id=user.id)
+    extraction_order._extraction_configuration = json.dumps(extraction_configuration_with_formats)
+    extraction_order.save()
+    extraction_order_id = extraction_order.id
+
+    new_apps = migrate('excerptexport', '0019_one_export_per_file_format')
+    ExtractionOrder = new_apps.get_model('excerptexport', 'ExtractionOrder')  # noqa
+    extraction_order_after_migration = ExtractionOrder.objects.get(id=extraction_order_id)
+    assert_that(
+        extraction_order_after_migration.exports.values_list('file_format', flat=True), contains_in_any_order(
```

Comment:
Dunno. Is
```python
    assert_that(
        extraction_order_after_migration.exports.values_list('file_format', flat=True),
        contains_in_any_order(
            'txt',
            'foobar',
        )
    )
```
more readable? If the line is too long to comfortably comprehend, it might make more sense to extract `extraction_order_after_migration.exports.values_list('file_format', flat=True)` to a local variable and to give that a good name.